controller/node_controller.go

```diff
@@ -266,6 +320,16 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 			diskStatus.StorageAvailable = diskInfo.StorageAvailable
 		}
 
+		// check disk pressure
+		if overProvisioningPercentage == 0 ||
```

Comment:
The calculation for `overProvisioningPercentage` and `minimalAvailablePercentage` should be already covered by the formula. Shouldn't need special case handling for them (except dividing zero if that happens).